sdk/src/vite/configPlugin.mts

```diff
@@ -75,6 +75,7 @@ export const configPlugin = ({
       build: {
         outDir: resolve(projectRootDir, "dist", "worker"),
         emitAssets: true,
+        ssrManifest: true,
```

Comment:
This approach looks really good! Great job 👏 

And this still indeed works with our Figtree font imported from CSS as expected. It's great that the test coverage has been steadily increasing in rwsdk 🚀